packages/shared/nodes/usfm/UsfmElementNode.ts

```diff
@@ -2,7 +2,7 @@ import { ElementNode, LexicalNode, NodeKey, SerializedElementNode, Spread } from
 
 export type Attributes = { [key: string]: string };
 
-export type SerializedUsfmElementNode = Spread<
+export type SerializedScriptureElementNode = Spread<
```

Comment:
Are you sure this should be renamed? It now clashes with another type of that same name. I would revert the 2 changes to this file if its not intentional.